### DISCUSSION TITLE
Conditional expression on variables

### DISCUSSION THREAD

**Discussion by Hennyr** [bitfocus/companion#2699] [Q&A]
Is it posible to use condition, line 'if ... then .. else' within trggers or feedback with variables?

**Comment by dnmeid** [comment]
You can use the ternary operator in expressions. It has the syntax of `a ? b : c` a is the condition, b is the returned statement if the condition is true and c is the returned statement if the condition is false. Example `$(custom:number) == 1 ? 'one' : 'not one'`.
That means the whole thing always returnes a value based on your condition, it is not meant to execute actions based on the condition. That is where you'd use a trigger. A trigger can have a condition and the actions of the trigger are executed when it is true, giving the 'if' part. For the 'else' partuse a second trigger and invert your condition or use 'on condition becoming false'
